Building with clang causes 'busybox less' to
segfault in INIT_G() on arm, see
https://github.com/termux/termux-packages/issues/1245

diff -u -r ../busybox-1.26.2/Makefile ./Makefile
--- ../busybox-1.26.2/Makefile	2017-01-10 17:01:16.000000000 +0100
+++ ./Makefile	2017-06-27 23:34:15.465534834 +0200
@@ -289,7 +289,7 @@
 # Make variables (CC, etc...)
 
 AS		= $(CROSS_COMPILE)as
-CC		= $(CROSS_COMPILE)gcc
+CC		?= $(CROSS_COMPILE)gcc
 LD		= $(CC) -nostdlib
 CPP		= $(CC) -E
 AR		= $(CROSS_COMPILE)ar
